src/HOCs/RenderByContextPerm.js

```diff
@@ -0,0 +1,30 @@
+import RenderByContextAuth from './RenderByContextAuth';
+
+/**
+ * Use these functions like this:
+ *
+ * OfficerRenderPermission( NameOfTheComponentToRender )( { a, b, c, ... } )
+ * MemberRenderPermission( NameOfTheComponentToRender )( { a, b, c, ... } )
+ * InducteeRenderPermission( NameOfTheComponentToRender )( { a, b, c, ... } )
+ *
```

Comment:
Can you give an example of rendering a component with props both with and without permissions wrapper?
Something like Component<a=True b=value> => xxRenderPermission(Component)({a: True, b:value})